SourceX/main.cpp

```diff
@@ -27,6 +31,17 @@ static std::string build_cmdline(int argc, char **argv)
 
 int main(int argc, char **argv)
 {
+#if defined(__3DS__)

```

Comment:
Contain this code in a function like was done with `switch_enable_network` so that main doesn't grow uncontrolably with each port.